src/EditorFeatures/TestUtilities/Utilities/TestWaitableUIOperationContext.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.UnitTests.Utilities
+{
+    internal class TestWaitableUIOperationContext : AbstractWaitableUIOperationContext
```

Comment:
Is this the editor-defined base type? Not sure "Abstract" should actually be in the type name...?